<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtnListar</class>
 <widget class="QWidget" name="vtnListar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Listar</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QComboBox" name="cbTipoListado">
     <item>
      <property name="text">
       <string>Productos en Stock</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Clientes</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ventas Realizadas</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbOption">
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="rbtnPDF">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>116</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtnExcel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>0</y>
        <width>116</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Planilla Excel</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbFechas">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Fecha Desde</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="deFechaDesde"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Fecha Hasta</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="deFechaHasta"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnListar">
       <property name="text">
        <string>Listar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
